tests/test_fixtures.py

```diff
@@ -30,6 +30,14 @@
             "M": 42,
             "GY": 44,
         },
+        "network": {
+            "Device Name": "EPSON0C9E89",
```

Comment:
I added compare for Device/Printer name. That should be unique. Maybe we could use this instead of having them enter a device name. Thoughts?

Reply:
I have locally tested using the Device/Printer name instead of the model number. 

<img width="665" height="322" alt="image" src="https://github.com/user-attachments/assets/e078efee-a0d4-4b69-8975-911b88b131c2" />

I do like the generated entity names better (especially if anyone had two of the same printers)

<img width="704" height="590" alt="image" src="https://github.com/user-attachments/assets/956ac8e7-e91c-467c-a497-45d8d6643b00" />

@alexbbt, what's your honest opinion?